mobile/lib/common/app_bar_wrapper.dart

```diff
@@ -17,9 +19,13 @@ class AppBarWrapper extends StatelessWidget {
     final currentRoute = GoRouterState.of(context).location;
     const appBarHeight = 35.0;
 
+    ChannelStatusNotifier channelStatusNotifier = context.watch<ChannelStatusNotifier>();
+
     var actionButtons = [
       IconButton(
-        icon: const Icon(Icons.thermostat),
+        icon: channelStatusNotifier.isClosing()
+            ? const Icon(Icons.thermostat, color: Colors.red)
+            : const Icon(Icons.thermostat),
```

Comment:
This is really cool ❤️! I'll make use of colours for other states when after this PR gets it.

Reply:
I'm glad you like it. I wasn't sure if it was gonna be received well haha.